packages/pipeline-editor/src/PipelineEditorWidget.tsx

```diff
@@ -116,7 +117,8 @@ export const commandIDs = {
   openMetadata: 'elyra-metadata:open',
   openDocManager: 'docmanager:open',
   newDocManager: 'docmanager:new-untitled',
-  submitNotebook: 'notebook:submit'
+  submitNotebook: 'notebook:submit',
+  addFileToPipeline: 'pipeline-editor:addToPipeline'
```

Comment:
minor nit, the commandID string shouldn't be camel case to match the standard in lab, also since the id is prepended with `pipeline-editor` a simpler id could be used
```suggestion
  addFileToPipeline: 'pipeline-editor:add-node'
```